src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -6257,10 +6257,10 @@ namespace SurfaceGeometry {
         using DataLoopNode::NodeConnectionType_Inlet;
         using DataLoopNode::NodeType_Air;
         using DataLoopNode::ObjectIsParent;
-        using DataSurfaces::SurfLocalEnvironment;
         using DataSurfaces::Surface;
-        using DataSurfaces::TotSurfLocalEnv;
+        using DataSurfaces::SurfLocalEnvironment;
```

Comment:
This looks completely unrelated. Should this file be touched at all? Same for  Vectors.cc

Reply:
Yeah, I was also confused with this diff. Clang moved this line to put in alphabetical order. Not sure why since I didn't change this file nor vectors.cc

Reply:
But if you didn't touch the file, clang shouldn't touch it either.